Applications/VpView/vpFileDataSource.cxx

```diff
@@ -154,21 +211,54 @@ void vpFileDataSource::update()
     {
       while (!file.atEnd())
       {
-        const auto line = file.readLine();
+        QString line = file.readLine().trimmed();
```

Comment:
Please don't decrease use of [AAA](https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/).